stable/node-local-dns/README.md

```diff
@@ -12,37 +12,11 @@ Further documentation is [here](https://kubernetes.io/docs/tasks/administer-clus
 
 This helm chart works for both kube-proxy setups (iptables or ipvs).
 
-## How to install this chart
```

Comment:
The tests are failing because you're removing all this. Do you need help with that?

Reply:
it's fixed, I was confused because I did run the automatic generation of readme but in the chart directory, not at the root of the repo